### PR TITLE
money dialog box

### PR DESCRIPTION
**Pull request by rahulv73039**
**Changes** 
1.Font Style Changed to Arial for text in dialog box.
2. Text Animation Removed.

**OutStandings**
1. I have Used game objecct GameObject For Initial Money  value.
2. I Have Made Animator Folder for animation of dialog box Up and Down Movement.

### CODE REVIEW

**Comment by mayant15** [comment]
You've committed your merge conflicts. Please test your PRs throroughly and make sure this doesn't happen in the future. Will fix and merge this one for now.

File: Assets/Scenes/MainScene.unity [hunk 1]
```diff
@@ -202,6 +1042,31 @@ PrefabInstance:
       objectReference: {fileID: 0}
     m_RemovedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: 284a761a17c45ee44b3075c871a586a1, type: 3}
+<<<<<<< HEAD
```
Comment (mayant15) [inline Assets/Scenes/MainScene.unity]: Merge conflict start marker

File: Assets/Scenes/MainScene.unity [hunk 2]
```diff
@@ -346,6 +1211,7 @@ PrefabInstance:
       objectReference: {fileID: 0}
     m_RemovedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: c3f33bd9d6e48cf43a6c01ebb8060a3f, type: 3}
+>>>>>>> c4be30b69c51fa199dcd7ae443f7f46b1f44d4cf
```
Comment (mayant15) [inline Assets/Scenes/MainScene.unity]: Merge conflict end marker

File: Assets/Scripts/UI/UpgradeMenuUISystem.cs [hunk 3]
```diff
@@ -1,6 +1,13 @@
 ﻿using UnityEngine;
+<<<<<<< HEAD
```
Comment (mayant15) [inline Assets/Scripts/UI/UpgradeMenuUISystem.cs]: Merge conflict start marker

File: Assets/Scripts/UI/UpgradeMenuUISystem.cs [hunk 4]
```diff
@@ -1,6 +1,13 @@
...
+using System.Collections.Generic;
+using System.Collections;
+using UnityEngine.UI;
+using System.Threading;
+=======
 using UnityEngine.UI;
 using System.Collections.Generic;
+>>>>>>> c4be30b69c51fa199dcd7ae443f7f46b1f44d4cf
```
Comment (mayant15) [inline Assets/Scripts/UI/UpgradeMenuUISystem.cs]: Merge conflict end marker

File: Assets/Scripts/UI/UpgradeMenuUISystem.cs [hunk 5]
```diff
@@ -14,9 +21,17 @@ public class UpgradeMenuUISystem : MonoBehaviour, IUISystem
     private GameObject greenPrefab;
     [SerializeField]
     private GameObject redPrefab;
+<<<<<<< HEAD
```
Comment (mayant15) [inline Assets/Scripts/UI/UpgradeMenuUISystem.cs]: Merge conflict start marker

File: Assets/Scripts/UI/UpgradeMenuUISystem.cs [hunk 6]
```diff
@@ -14,9 +21,17 @@ public class UpgradeMenuUISystem : MonoBehaviour, IUISystem
...
+    public Animator animator;
+=======
     private List<GameObject> upgradeButtons = new List<GameObject>();
+>>>>>>> c4be30b69c51fa199dcd7ae443f7f46b1f44d4cf
```
Comment (mayant15) [inline Assets/Scripts/UI/UpgradeMenuUISystem.cs]: Merge conflict end marker

**Review by mayant15 — CHANGES_REQUESTED** [review]
You've committed all merge conflicts. Fix it and ensure this doesn't happen in the future. Test your PRs thoroughly. We can discuss how to solve this tonight 👍

**Comment by rahulv73039** [comment]
i think conflicts are resolved 
In upgrademenu script it was just displacement of text and nothing more  complex
In main scene there was conflicts of adresses i think

File: Assets/Scripts/Dialogbox/scoreup.cs [hunk 7]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class scoreup : MonoBehaviour
+{
+    public GameObject scoreText;
+    public  int theScore;
```
Comment (mayant15) [inline Assets/Scripts/Dialogbox/scoreup.cs:9]: Use better variable names, like just `score`

File: Assets/Scripts/UI/UpgradeMenuUISystem.cs [hunk 8]
```diff
@@ -60,15 +72,63 @@ public void OnClick(string type)
...
         {
             case "green":
-                CreateNewTower(greenPrefab);
+                if (gameobject.GetComponent<scoreup>().theScore >= 20)
+                {
+                    animator.SetBool("IsOpen", false);
+                    gameobject.GetComponent<scoreup>().theScore -= 20;
+                    
+                    CreateNewTower(greenPrefab);
+
```
Comment (mayant15) [inline Assets/Scripts/UI/UpgradeMenuUISystem.cs:81]: Remove unnecessary blank lines like this one

File: Assets/Tests/PlayMode/MovementTests.cs [hunk 9]
```diff
@@ -31,5 +31,30 @@ public IEnumerator AIMovePlayer()
             yield return new WaitForSeconds(1f);
             Assert.AreNotEqual(initial, player.transform.position);
         }
+        /// <summary>
```
Comment (mayant15) [inline Assets/Tests/PlayMode/MovementTests.cs:34]: These tests seem to be left-overs from your earlier PRs

**Review by mayant15 — APPROVED** [review]
Apart from a few Git mistakes, the PR seems complete. Will clean it up a bit and merge today.

**Pull request merged**